mcs/class/System.Net.Http/Makefile

```diff
@@ -38,7 +38,7 @@ LIB_MCS_FLAGS += -d:XAMARIN_MODERN
 endif
 
 ifeq (wasm,$(PROFILE))
-API_BIN_REFS := WebAssembly.Net.Http
+API_BIN_REFS := System.Net.Http.WebAssemblyMessageHandler
```

Comment:
I think this needs to change to be reference the files again.

@lewing @kjpou1 the thinking is to have the implementation in 2 assemblies
- System.Net.Http.WebAssemblyHttpHandler (nuget exposing WebAssembly specific API)
- System.Net.Http - internally calling into System.Net.Http.WebAssemblyHttpHandler and not exposing any additional API.

Could you help Pranav to land this change